locales/templates/app.pot

```diff
@@ -969,3 +974,63 @@ msgctxt "tab_tray_close_tabs_dialog_msg"
 msgid "Close all tabs?"
 msgstr ""
 
+#. This is the title of Themeing on-boarding dialog
+msgctxt "themetoy_onboarding_title"
+msgid "Color your life!"
+msgstr ""
+
+#. This is the description of Themeing on-boarding dialog
+msgctxt "themetoy_onboarding_description"
+msgid "Double click on the home background to switch themes."
+msgstr ""
+
+#. Message shown in a SnackBar after a bookmark is saved.
```

Comment:
We should try to refrain from using technical jargon such as "SnackBar" and "Toast" from localization comments: since they are more developer-oriented, many translators won't get it (or will imagine this: 
![2cp85q](https://user-images.githubusercontent.com/399020/41785053-478ac706-75f6-11e8-8961-c5b9604b48c9.jpg)